docs/content/docs/dev/configuration/connector.md

```diff
@@ -24,39 +24,42 @@ under the License.
 
 # Connectors and Formats
 
-Flink can read from and write to various external systems via connectors and define the format in 
-which to store the data.
+Flink can read from and write to various external systems via connectors and use the format of your choice
+in order to read/write data from/into records.
 
-The way that information is serialized is represented in the external system and that system needs
```

Comment:
Imho, we should also keep this paragraph.

Reply:
Not insisting, let's also hear other's opinions, but I'd like this 2 sentence extra explanation about the formats, even if it's explained in detail in the provided link.

Reply:
My proposal would be:

```
Flink applications can read from and write to various external systems via connectors. 
It supports multiple formats in order to encode and decode data to match Flink's data structures.
```

Reply:
Updated as you suggested @MartijnVisser